src/modules/85-cv/pages/health-source/connectors/AppDynamics/AppDHealthSource.tsx

```diff
@@ -251,7 +251,7 @@ export default function AppDMonitoredSource({
       initialValues={initPayload}
       onSubmit={noop}
     >
-      {formik => {
+      {(formik: any) => {
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9232293)
```suggestion
      {(formik: unknown) => {
```